code/modules/projectiles/projectile/energy.dm

```diff
@@ -67,7 +67,7 @@
 /obj/item/projectile/energy/electrode/stunshot
 	nodamage = 0
 	damage = 10
-	agony = 80
+	agony = 50

```

Comment:
/obj/item/projectile/beam/stun
Which deals 40 pain and downs someone in an average of three shots not counting armor.
Unless you headshot them, in that case it's around 2